frontend/console-extensions.json

```diff
@@ -0,0 +1,200 @@
+[
+    {
+        "type": "console.perspective",
+        "properties": {
+            "id": "acm",
+            "name": "All clusters",
+            "icon": { "$codeRef": "perspective.icon" },
+            "landingPageURL": { "$codeRef": "perspective.getLandingPageURL" },
+            "importRedirectURL": { "$codeRef": "perspective.getImportRedirectURL" }
```

Comment:
What is this used for? Found it documented as "The function to get redirect URL for import flow" but I am not sure what that means.

Reply:
not sure about the import...  it is required so i just put in that the demo had